tamr_unify_client/project/attribute_mapping_collection.py

```diff
@@ -0,0 +1,30 @@
+import requests
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+from tamr_unify_client.project.attribute_mapping import AttributeMapping
+
+
+class AttributeMappingCollection(AttributeMapping):
+    def setUp(self):
```

Comment:
`setUp` comes from the `TestCase` class, we shouldn't have it here. There should be an `init` though, which should take in the API path (the other functions should not)